collections/_article/beyond-100000-you-re-breathtaking.md

```diff
@@ -0,0 +1,100 @@
+---
+title: "Beyond #100000: You're breathtaking!"
+excerpt: "As we reach issue #100000 on GitHub, we celebrate with a little retrospective on our issue and PR growth."
+categories: ["news"]
+author: "Rémi Verschelde"
+image: /storage/blog/covers/beyond-100000-you-re-breathtaking.webp
+image_caption_title: "\"You're breathtaking!\""
+image_caption_description: "Sketch of Keanu Reeves at the 2019 #XboxE3 event"
+date: 2024-12-04 19:00:00
+---
+
+<div markdown=1 class="card card-info" style="margin-top: 1em;">
+This blog post has been originally posted on [GH-100000](https://github.com/godotengine/godot/issues/100000).
+</div>
+
+It's been over 10 years now that Godot has been developed in the open, with code contributions from 2,800 users.
+
+The total count of people who helped build Godot is actually far greater, as it should include documentation writers, testers, bug reporters, translators, moderators, content creators, users doing community support or talking at events, folks making games that credit Godot visibly, everyone supporting the project financially, and many other types of contributions which I can't keep enumerating.
+
+All these people brought Juan and Ariel's little-engine-that-could from this:
+
+![Screenshot of the isometric 2D demo in Godot 1.0](/storage/blog/beyond-100000-you-re-breathtaking/godot-1-0.webp)
+<div markdown=1 class="small text-center">
+*Screenshot of the isometric 2D demo in Godot 1.0.*
+</div>
+
+To this:
+
+![Screenshot of PVKK: Planetenverteidigungskanonenkommandant in Godot 4.3](/storage/blog/beyond-100000-you-re-breathtaking/godot-4-3.webp)
+<div markdown=1 class="small text-center">
+*Screenshot of [PVKK: Planetenverteidigungskanonenkommandant](https://store.steampowered.com/app/2956040/PVKK_Planetenverteidigungskanonenkommandant/) in Godot 4.3.*
+</div>
+
+That's no small achievement, so I encourage all contributors to take a minute to contemplate the progress that we've made together over this journey so far!
+
+Amidst the daily churn of fixing issues, reviewing PRs, making releases, etc., it's important to remind ourselves of where we are, and how we got there.
```

Comment:
```suggestion
Amidst the daily churn of fixing issues, reviewing pull requests, making releases, etc., it's important to remind ourselves of where we are, and how we got there.
```

Reply:
I wrapped all PR and PRs mentions with <abbr>. It looks like this.

<img width="840" alt="Capture d’écran, le 2024-12-04 à 15 29 10" src="https://github.com/user-attachments/assets/46dad9fc-e6a6-41a1-8e1e-2b8718a1adba">